### ISSUE TITLE
[Arcs] Conspirator declares an ambition

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#66]
Is the Conspirator still declaring an ambition even though they don't place an ambition marker?

**Comment by turmoilbyrd** [comment]
First mentioned in https://github.com/LederCards/cards/issues/60

**Comment by turmoilbyrd** [comment]
My opinion is that it is, even though the wording makes it sound a bit like a "replacement effect" (since it contains the word "instead")

**Comment by jdyearsley** [comment]
Yes, as written they are still declaring an ambition. I'll add an FAQ to the Conspiracies card that says you still trigger Farseers, are restricted by the Believer's Faith, etc.
